public/apps/configuration/app-router.tsx

```diff
@@ -13,8 +13,8 @@
  *   permissions and limitations under the License.
  */
 
-import { EuiBreadcrumb, EuiPage, EuiPageBody, EuiPageSideBar } from '@elastic/eui';
-import { flow, partial } from 'lodash';
+import { EuiPage, EuiPageBody, EuiPageSideBar } from '@elastic/eui';
```

Comment:
Nit: Can these be updated to the newer Oui assets?

Reply:
OSD aliases Oui as Eui, so we are good.